src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,45 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+          %h4
+            Reviews
+          .mb-4
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: accepted_reviews, as: :review
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: declined_reviews, as: :review
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: open_reviews, as: :review
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+        - if open_reviews_for_staging_projects.present?
+          %h4
+            Reviews for staging projects
```

Comment:
Reviews for staging projects are not really reviews. They are "just" the means to tell people that their request is staged and where. If there is one present then it means the request is staged the staging project. Don't make it implicit again.

Reply:
I should've listened to @rubhanazeem :facepalm: Changed again. Now it looks like this again:

![Screenshot 2022-08-10 at 14-01-37 Open Build Service](https://user-images.githubusercontent.com/2581944/183896104-48913687-7264-4944-ad92-526b287d7afe.png)